src/plugins/event-plugins/XhrPlugin.ts

```diff
@@ -244,11 +259,14 @@ export class XhrPlugin extends MonkeyPatched<XMLHttpRequest, 'send' | 'open'> {
         xhr: XMLHttpRequest
     ) {
         if (this.config.recordAllRequests || !this.statusOk(xhr.status)) {
-            this.context.record(HTTP_EVENT_TYPE, {
+            const httpEvent: HttpEvent = {
```

Comment:
nit: There are a lot of minor stylistic changes in this CR (e.g., creating a temporary variables, renaming variables, re-arranging imports). These changes don't add value to the package, and make it more difficult to review.

Also keep in mind that saving bytes matters in this package. The packager/minimizer can only do so much, so adding additional statements and long variable names can increase the size of the package.

Reply:
updated to remove temp vars and renamings